controllers/apply/awards-for-all/fields.js

```diff
@@ -890,6 +984,13 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                         en: `Enter your organisation’s Department for Education number`,
                         cy: ''
                     })
+                },
+                {
+                    type: 'string.max',
+                    message: localise({
+                        en: `Organisation’s Department for Education number must be ${FREE_TEXT_MAXLENGTH.large} characters or less`,
```

Comment:
'Department for Education number must be ${FREE_TEXT_MAXLENGTH.large} characters or less`